styles.css

```diff
@@ -0,0 +1,335 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background-color: white;
+  font-family: 'Karla', sans-serif;
+}
+
+h1 {
+  color: #2B547E;
+}
+
+h2 {
+  color: #2B547E;
+
+}
+
+.header-container {
+  background-image: url("images/header-image.jpg");
+  background-repeat: no-repeat;
+  display: flex;
+  background-color: grey;
+  width: 100%;
+  height: 30rem;
+  position: relative;
+}
+
+.header-logo {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin-left: 50px;
+  width: 20%;
+
+}
+
+.header-container p {
+  color: #81B741;
+  font-size: 1.2rem;
+  display: flex;
+  flex-wrap: wrap;
+  width: 100%;
+}
+
+
+
+.cart-container {
+  display: flex;
+
+}
+
+
+.cart-icon {
+  display: flex;
+  width: 2rem;
+  position: absolute;
+  top: 5%;
+  right: 10%;
+  color: black;
+
+
+}
+
+.checkout-button {
+  width: 2rem;
+  position: absolute;
+  top: 15%;
+  right: 15%;
+  padding: 1rem;
+  font-size: bold;
+}
+
+
+.nav {
+  background-color: #D3D3D3;
+  width: 100%;
+  height: 3rem;
+
+}
+
+  .nav ul li  {
+   list-style: none;
+   display: inline;
+   margin-left: 5%;
+   margin-top: 5%;
+   color: white;
+   padding: 2rem;
+   padding-top: 3rem;
```

Comment:
the padding on these elements causes the elements to be bigger than the container. Maybe instead style the .nav with padding, you might only need margin left on these to lay them out right.